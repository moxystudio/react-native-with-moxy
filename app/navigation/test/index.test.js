import React from 'react';
import Navigation from '../';
import { shallow } from 'enzyme';

it('should create navigation correctly', () => {
    const tree = shallow(<Navigation />);

    expect(tree).toMatchSnapshot();
});

