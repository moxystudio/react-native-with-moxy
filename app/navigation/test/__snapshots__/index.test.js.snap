// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should create navigation correctly 1`] = `
<StackNavigator>
  <Screen
    component={[Function]}
    name="Home"
  />
</StackNavigator>
`;
